src/lib/commandBarConfigs/authCommandConfig.ts

```diff
@@ -21,7 +22,8 @@ export function createAuthCommands({
       groupId: ACTOR_IDS.AUTH,
       name: 'refresh',
       displayName: 'Refresh app',
-      icon: 'arrowRotateRight',
+      description: 'Force the the scene, features, and editor to reload.',
+      icon: 'exclamationMark',
```

Comment:
This matches the icon in the sidebar that @franknoirot changed in https://github.com/KittyCAD/modeling-app/pull/7682.